src/terminal/parser/precomp.h

```diff
@@ -10,6 +10,9 @@ Module Name:
 - Avoid including internal project headers. Instead include them only in the classes that need them (helps with test project building).
 */
 
+#include <winrt/Windows.Security.Cryptography.h>

```

Comment:
Okay I figured it out!

You need to include `<wil/cppwinrt.h>` _after_ `wil`, but before the rest of the winrt headers. So change this file to the following:

```c++
#include "LibraryIncludes.h"

#include <windows.h>

#include <sal.h>

#include <cstdio>

#define ENABLE_INTSAFE_SIGNED_FUNCTIONS
#include <intsafe.h>

#include "telemetry.hpp"
#include "tracing.hpp"

#include <wil/cppwinrt.h>
#include <winrt/Windows.Security.Cryptography.h>
#include <winrt/Windows.Storage.Streams.h>

```

Reply:
Works like a charm. Cheers!